[2025-06-13T22:57:41.268+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_projeto.load_dw manual__2025-06-13T22:57:25.678857+00:00 [queued]>
[2025-06-13T22:57:41.278+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_projeto.load_dw manual__2025-06-13T22:57:25.678857+00:00 [queued]>
[2025-06-13T22:57:41.279+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-06-13T22:57:41.293+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_dw> on 2025-06-13 22:57:25.678857+00:00
[2025-06-13T22:57:41.298+0000] {standard_task_runner.py:60} INFO - Started process 200 to run task
[2025-06-13T22:57:41.302+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'etl_projeto', 'load_dw', 'manual__2025-06-13T22:57:25.678857+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpsjl4r9nl']
[2025-06-13T22:57:41.304+0000] {standard_task_runner.py:88} INFO - Job 44: Subtask load_dw
[2025-06-13T22:57:41.368+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_projeto.load_dw manual__2025-06-13T22:57:25.678857+00:00 [running]> on host a984848cc8d0
[2025-06-13T22:57:41.476+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kayo' AIRFLOW_CTX_DAG_ID='etl_projeto' AIRFLOW_CTX_TASK_ID='load_dw' AIRFLOW_CTX_EXECUTION_DATE='2025-06-13T22:57:25.678857+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-13T22:57:25.678857+00:00'
[2025-06-13T22:57:42.341+0000] {logging_mixin.py:188} INFO - Carga no DW finalizada!
[2025-06-13T22:57:42.342+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-13T22:57:42.359+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_projeto, task_id=load_dw, execution_date=20250613T225725, start_date=20250613T225741, end_date=20250613T225742
[2025-06-13T22:57:42.397+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-13T22:57:42.420+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
